ex3/ex3.js

```diff
@@ -7,3 +7,62 @@
  * containsValue([1,3,5,7,9], 7) === true;
  * containsValue([9,7,5,3,1], 7) === true;
  */
+function containsValue(arrayNumber, searchForNumber) {
```

Comment:
arrayNumbers is probably a better name.
Also, keep in mind that my question didn't deal with just numbers (only in the example) and would also work for something like: containsValue(["a", "b", "c"], "d") === false. So, you shouldn't necessarily assume this **only** deals with numbers.